pyogrio/geopandas.py

```diff
@@ -59,6 +60,10 @@ def read_dataframe(
     bbox : tuple of (xmin, ymin, xmax, ymax) (default: None)
         If present, will be used to filter records whose geometry intersects this
         box.  This must be in the same CRS as the dataset.
+    fids : array-like, optional (default: None)
+        Array of integer feature id (FID) values to select. Cannot be combined
+        with other keywords to select a subset (`skip_features`, `max_features`,
+        `where` or `bbox`).
```

Comment:
```suggestion
        `where` or `bbox`).  Note that the starting index is driver specific.
```
(just a nudge to users to remember that GPKG starts at 1, others at 0)

Reply:
Not sure if it is also worth mentioning here that the performance of reading large numbers of features using FIDs is driver specific, and may be much worse than reading the entire dataframe and selecting out features of interest afterward.

Reply:
I expanded the note a bit, please take a look (I kept the performance note short for now, because actually starting to explain this might get a bit long ..).